### PR TITLE
Manipulation is  focus object only

### PR DESCRIPTION
**Pull request by kaorun55**

### CODE REVIEW

**Comment by jbienzms** [comment]
I'm not entirely sure I understand the issue, based on your description and code alone. Can you elaborate on the purpose of this change please? Thanks kaorun55!

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,7 @@ Library/
 Assets/AssetStoreTools*
 App/
 UWP/
+WindowsStoreApp/
```
Comment (NeerajW) [inline .gitignore]: Does this need to be a public addition? Please feel free to revert.

Comment (NeerajW) [inline .gitignore]: Also please revert templates.meta. You don't need any changes in those.

File: ProjectSettings/EditorBuildSettings.asset [hunk 2]
```diff
@@ -4,4 +4,6 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
```
Comment (NeerajW) [inline ProjectSettings/EditorBuildSettings.asset]: Please revert this file. Project level build settings don't need this update.

**Comment by NeerajW** [comment]
@jbienzms if I read the code right, @kaorun55 is simply ensuring that the GestureManipulator script is indeed on the focused game object. OR making sure that the GestureManipulator logic will kick in only for the game object that is in focus and not for every game object.

**Comment by micthom-msft** [comment]
The current implementation _intentionally_ does not tie itself to the FocusedObject.  The idea was to create an implementation of Hand Manipulation that could be easily consumed by anyone, regardless of whether they are using the GazeManager/Object Focus system.  If we want to change that I would love to have a specific conversation about the pros/cons (we should probably open an issue) before making any changes.

**Comment by kaorun55** [comment]
Sorry.
I changed GestureManipulator.cs,GestureManipulator.unity.

In the code of the up to now had to move all of GameObject. A modification will be moved only FocusObject.

**Comment by kaorun55** [comment]
WindowsStoreApp add .gitignore(This PR revert).
HoloToolkit / Build Window of Build directory is "WindowsStoreApp".
Is this do not need?

**Comment by NeerajW** [comment]
@kaorun55 per @micthom-msft's request please file an issue for this change and let's have a design discussion first about this pull request.

**Comment by HodgsonSDAS** [comment]
It's been two weeks and I think we all agreed this wasn't a good change.
No CLA has been signed, can we please close this PR?

**Pull request closed without merging**